client-registry-jempi/importer/jempi_psql_init_db.sql

```diff
@@ -0,0 +1,80 @@
+CREATE TABLE IF NOT EXISTS Notification_Type
```

Comment:
If the backend of JeMPI requires these tables to exist, then it should be responsible for creating these tables. This should be done in a migration, not as part of a docker config. 

@rcrichton Please provide your input here, I'm not sure if it makes sense for us to have this SQL script in Platform as that means we are responsible for its maintenance based on developments that take place outside of our control. Not sure who to speak to about it.

Reply:
This is how Jempi is done now, maybe they can create their own image of postgres and use that but now there is no such image

Reply:
We will roll with this method for now, but let's have a discussion around this in Arch forum on Wednesday.